subsys/bluetooth/audio/bap_stream.h

```diff
@@ -7,6 +7,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+void bt_bap_stream_init(struct bt_bap_stream *stream);
```

Comment:
> Since this is only used by ASCS, why not keep it as a `static` function?
I would like to avoid increasing dependency between modules in the code. It shall be `bt_bap_stream` interface function.

> Or do you intend on using this for the other roles too?
Not now. I guess it can be reused without a need for any modifications is it